packages/plugin-init/sources/commands/init.ts

```diff
@@ -207,7 +206,7 @@ export default class InitCommand extends BaseCommand {
         },
       };
 
-      merge(editorConfigProperties, configuration.get(`initEditorConfig`));
+      miscUtils.mergeIntoTarget(editorConfigProperties, configuration.get(`initEditorConfig`));
```

Comment:
Why do we need these utils? Does the yaml stringifier supports serializing the comments?